executor/common_bsd.h

```diff
@@ -278,3 +278,68 @@ static int do_sandbox_none(void)
 	return 0;
 }
 #endif // SYZ_EXECUTOR || SYZ_SANDBOX_NONE
+
+#if SYZ_EXECUTOR || SYZ_SANDBOX_SETUID
+
+#include <sys/resource.h>
+#include <sys/wait.h>
+#include <unistd.h>
+
+static void sandbox_common()
+{
+	if (setsid() == -1)
+		fail("setsid failed");
+
+	struct rlimit rlim;
+#ifndef GOOS_openbsd
+	// Documented bug in OpenBSD.
+	rlim.rlim_cur = rlim.rlim_max = 200 << 20;
+	setrlimit(RLIMIT_AS, &rlim);
+#endif
+	rlim.rlim_cur = rlim.rlim_max = 32 << 20;
```

Comment:
We already do this in executor_bsd.h:os_init.
But there is a TODO explaining why that is a wrong place for that code.
If you call sandbox_common from do_sandbox_none, it should resolve that TODO as well.
However, I am not sure about setsid call, because we did not do it before.

Reply:
> We already do this in executor_bsd.h:os_init.
> But there is a TODO explaining why that is a wrong place for that code.

Moved.

> If you call sandbox_common from do_sandbox_none, it should resolve that TODO as well.
> However, I am not sure about setsid call, because we did not do it before.

I'm inclined to blame it on a simple omission. {Free,Net,Open}BSD all seem to have similar setsid.